gulpfile.js

```diff
@@ -1,34 +1,38 @@
-var gulp = require('gulp');
-var gutil = require('gulp-util');
-var webpack = require('webpack');
+const gulp = require('gulp');
+const log = require('fancy-log');
```

Comment:
Just curious, why did you replace `gutil` with `log` here but not in Encode?

Reply:
I do not remember the detail. gutil is deprecated (https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5). So eventually should be replaced. I can replace it in ENCODE if you desire.

Reply:
It might be a good time to replace it in Encode if we'll have to replace it eventually.

Reply:
Replaced.